CHANGELOG.md

```diff
@@ -19,13 +19,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Support for OpenGL ES 2.0
 - Escape sequence to set underline color (`CSI 58 : 2 : Ps : Ps : Ps m`/`CSI 58 : 5 : Ps m`)
 - Escape sequence to reset underline color (`CSI 59 m`)
-- Vi mode keybinding (z) to center view around vi mode cursor 
+- Vi mode keybinding (z) to center view around vi mode cursor
+- `WINIT_WAYLAND_CSD_THEME` env variable to control decorations "dark"/"light" variant
+- Support for inline input method
 
 ### Changed
 
 - The `--help` output was reworked with a new colorful syntax
 - OSC 52 is now disabled on unfocused windows
 - `SpawnNewInstance` no longer inherits initial `--command`
+- Client side decorations should have proper text rendering now on Wayland
+- Search bar is now respecting cursor thickness
```

Comment:
Not sure this should be fixed in this PR?

Reply:
It's a part of IME commit.